client/state/comments/reducer.js

```diff
@@ -216,4 +249,6 @@ export default combineReducers( {
 	fetchStatus,
 	errors,
 	totalCommentsCount,
+	trees,
+	counts,
 } );
```

Comment:
the end-result of using `keyedReducer` is further composition keyed by the given property.

```js
export const counts = ( state = 0, { count, type } ) =>
	COMMENTS_TREE_ADD === type
		? count
		: state;

export const commentsTreeItem = combineReducers( {
	count,
	tree,
} );

export const commentsTree = keyedReducer( 'siteId', commentsTreeItem );
```